src/bloqade/submission/braket.py

```diff
@@ -53,20 +78,69 @@ def get_capabilities(self, use_experimental: bool = False) -> QuEraCapabilities:
         return super().get_capabilities()
 
     def submit_task(self, task_ir: QuEraTaskSpecification) -> str:
+        """Submit the task to the Braket backend. It converts task
+        IR(Intermediate Representation) of the QuEra system to suitable format
```

Comment:
I think its fine to leave out the `(ntermediate Representation)` here this is mostly an internal function for developers to work with.